src/assets/scss/RedeemButtons.scss

```diff
@@ -0,0 +1,30 @@
+.hoverButtons{
```

Comment:
Usually, it is recommended to have space before the opening curly braces when defining your class names or id names in CSS